settings.gradle

```diff
@@ -39,5 +39,8 @@ pluginManagement {
 }
 
 rootProject.name = 'rxdogtag-root'
+if (System.getenv("ANDROID_HOME") != null) {
```

Comment:
Similar to the JMH benchmarks, maybe we should include a little snippet on how to run the Android benchmarks somewhere?

Reply:
Good call, added here - b1c832c

Will flesh out a full wiki after both of these are in as well